frontend/src/theme/colors.ts

```diff
@@ -0,0 +1,40 @@
+const colors = {
```

Comment:
Not sure what color system is conducive to clear design<>dev mappings. `vividBlue` is more clear in naming but the `blue` object is  modifying the `blue` color scheme itself - i.e. `colorScheme="blue"` button props (`StoryCard.tsx`)
@ZhangCreations curious what our experience developing with colour was like in chakra ui

Reply:
In the past, I've worked with a much simpler colour scheme with fewer colours so we could name things like "background" and "titles" etc.  If we coordinate with design on numbers for each tone i.e blue.100 the designers can note exactly which shade they're using in Figma which would be pretty fluid for devs.

Reply:
Good suggestion, sent a message in #planetread-design about it